mastercraft.cmd

```diff
@@ -485,7 +500,7 @@ turn.page:
      gosub GET my %discipline book
      action (book) on
      gosub PUT turn my book to chapter %order.chapter
-     send read my book
+     put read my book

```

Comment:
In the following, you will see a lot of "send read my book" replaced with put read my book.
I was going to gosub put read my book, but there is not appropriate handling for that. 
I am in the process of writing out handling. Really what is happening is that the script will hang here during laggy times. I will not comment any more of the lines that match this pattern